test/wallet-test.js

```diff
@@ -2114,7 +2116,7 @@ describe('Wallet', function() {
     async function mineBlock(tip) {
       const job = await miner.createJob(tip);
       const block = await job.mineAsync();
-      chain.add(block);
+      return chain.add(block);
```

Comment:
Good catch here, test fails without it or `await`, the function returns a promise we need to wait for.

Reply:
Yep